src/js/organizer.client.jsx

```diff
@@ -21,27 +19,22 @@ require('!style!css!sass!../sass/main.scss');
 
 const app = document.getElementById('app');
 
-const auth = new AuthService(config.clientId, config.clientDomain);
-
 // onEnter callback to validate authentication in private routes
 const requireAuth = (nextState, replace) => {
-  if (!auth.loggedIn()) {
+  if (!AuthService.loggedIn()) {
     replace({ nextPathname: nextState.location.pathname }, '/login');
   }
 };
 
 ReactDOM.render(
```

Comment:
@larajanecka is there a reason this rerouting differs so much from the attendee app?